includes/site-health/audit-autoloaded-options/helper.php

```diff
@@ -202,23 +196,24 @@ function perflab_aao_get_autoloaded_options_table() {
  * @return string HTML formatted table.
  */
 function perflab_aao_get_disabled_autoloaded_options_table() {
-	global $wpdb;
-
 	$disabled_options = get_option( 'perflab_aao_disabled_options', array() );
 
 	if ( empty( $disabled_options ) ) {
 		return '';
 	}
 
-	$disabled_options_summary = $wpdb->get_results(
-		$wpdb->prepare(
-			sprintf(
-				"SELECT option_name, LENGTH(option_value) AS option_value_length FROM $wpdb->options WHERE option_name IN (%s) ORDER BY option_value_length DESC",
-				implode( ',', array_fill( 0, count( $disabled_options ), '%s' ) )
-			),
-			$disabled_options
-		)
-	);
+	$disabled_options_summary = array();
+
+	foreach ( $disabled_options as $option_name ) {
+		$option_value = get_option( $option_name );
```

Comment:
Prior to making so many calls to `get_option()` we could run all of the disabled options through `wp_prime_option_caches()` to avoid multiple DB queries.

Reply:
thanks, commit done